### PR TITLE
E2E tests stabilization and traces tests 

### PR DESCRIPTION
**Pull request by ayuwono**
Pardon me for the mess and back and forth push as I need to have trial and error tests on jenkins env.

This PR has the E2E tests stabilized. It used a different method to write custom content on content.sjs plugin, it seems that it's the failures. I put some run on linux jenkins sandbox and it seems that it's stable:
https://jenkins.marklogic.com/view/Labs/job/Labs/job/DHF_e2e_QuickStart_ayuwono_sandbox_b9_nightly_linux/
It's passing consistently on mac env too

The one failure is expected as it's a bug (logged issue #746)

Added some preliminary tests on traces page (the tests will be expanded later)

### CODE REVIEW

File: quick-start/e2e/specs/index.ts [hunk 1]
```diff
@@ -44,7 +45,8 @@ describe('QuickStart', function () {
       // our Jenkins machine runs with a pretty low resolution, and we also
       // have an app that's misbehaving in smaller windows, so this is a delicate
       // setting
-      .then(() => browser.driver.manage().window().setSize(width, height))
+      //.then(() => browser.driver.manage().window().setSize(width, height))
```
Comment (dmcassel) [inline quick-start/e2e/specs/index.ts]: please remove commented-out line

File: quick-start/protractor.conf.js [hunk 2]
```diff
@@ -14,20 +14,20 @@ exports.config = {
   capabilities: {
     'browserName': 'chrome',
     chromeOptions: {
-      args: ["--headless", "--disable-gpu", "--window-size=1920,1080" ]
-      //args: ["--window-size=1600,1000"]
+      args: ["--headless", "--disable-gpu", "--window-size=1920,1080"]
```
Comment (dmcassel) [inline quick-start/protractor.conf.js]: are these numbers visible to e2e/specs/index.ts? possible to import them there and have them in just one place?

File: quick-start/protractor.conf.js [hunk 3]
```diff
@@ -14,20 +14,20 @@ exports.config = {
...
+      //args: ["--window-size=1920,1080"]
```
Comment (dmcassel) [inline quick-start/protractor.conf.js]: remove commented-out line

File: quick-start/protractor.conf.js (hunk 2 above)
Comment (ayuwono) [reply quick-start/protractor.conf.js]: Commented lines are removed

**Comment by dmcassel** [comment]
Saw three failures running the tests on Windows: 

```
**************************************************
*                    Failures                    *
**************************************************

1) QuickStart auth login Has the correct current folder
  - Expected 'C:\Users\dcassel\git\marklogic-data-hub\quick-start' to contain '/quick-start'.

2) QuickStart runFlows Run Flows should run Harmonize Products flow
  - Failed: No element found using locator: By(css selector, #flows-tab)

3) QuickStart jobs Run Jobs search only input jobs
  - Expected 'Showing Results 1 to 6 of 6' to contain 'Showing Results 1 to 5 of 5'.
```

First one is a platform difference, should be easy to fix. Not sure about the other two.

**Comment by dmcassel** [comment]
more changes pushed, rerunning tests

**Pull request merged**